.github/workflows/android-perf-test-firebase.yml

```diff
@@ -0,0 +1,42 @@
+name: Android Performance Tests in Firebase
```

Comment:
@jjSDET As we talked yesterday, let's create a PR just with this github action so that we can test it with the changes in this PR. Thanks